tests/qgis-projects/tests/test_cache.md

```diff
@@ -1,10 +1,23 @@
 This test is to check the cache.
 - [ ] Config the tile cache as file cache
-- [ ] Create the the cache with the lizmap command line:
+- [ ] Create a part of the cache with the lizmap command line:
   - `php lizmap/scripts/script.php lizmap~wmts:capabilities testsrepository test_cache`
   - `php lizmap/scripts/script.php lizmap~wmts:capabilities -v  testsrepository test_cache Quartiers EPSG:3857`
-  - `php lizmap/scripts/script.php lizmap~wmts:seeding -v -f testsrepository test_cache Quartiers EPSG:3857 0 18`
+  - `php lizmap/scripts/script.php lizmap~wmts:seeding -v -f testsrepository test_cache Quartiers EPSG:3857 10 10`
 - [ ] Check the tile cache size
+  - `du /tmp/testsrepository/test_cache/Quartiers/EPSG_3857/lizmap_/`
```

Comment:
You could use `du -sc` to have a total given in one line cf `man du`

```
       -c, --total
              produce a grand total
[...]
       -s, --summarize
              display only a total for each argument

```

Reply:
Done thanks